reana_workflow_engine_yadage/externalbackend.py

```diff
@@ -88,6 +88,9 @@ def set_parameter(resource: Dict[str, Any], key: str) -> None:
             set_parameter(item, "htcondor_accounting_group")
             set_parameter(item, "slurm_partition")
             set_parameter(item, "slurm_time")
+            set_parameter(item, "c4p_cpu_cores")
+        set_parameter(item, "c4p_memory_limit")
+        set_parameter(item, "c4p_additional_requirements")
```

Comment:
I think the last two calls to `set_parameter` are incorrectly indented?

Reply:
Thanks for the hint. I have changed the indentation.